pkgs/development/python-modules/python-imports-check-libstdcxx/main.cc

```diff
@@ -0,0 +1,51 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include <condition_variable>
+#include <mutex>
+#include <thread>
+
+#ifndef LOAD_LIBSTDCXX_MODULE_NAME
+#error Missing -DLOAD_LIBSTDCXX_MODULE_NAME
+#endif
+
+PYBIND11_MODULE(LOAD_LIBSTDCXX_MODULE_NAME, m) {
+  m.doc() = R"pbdoc(
+        This module doesn't do anything,
+        it only makes the python interpreter load a libstdc++.
+        Nixpkgs uses this to ensure that their python packages do not segfault just because of order of imports
+    )pbdoc";
+
+  // Make sure we actually require some symbols from libstdc++
+  // E.g. earlier we've observed the combination of [ "torch" "zmq" ]  fail
+  // because of std::condition_variable
+  m.def("dummy", []() {
+    // Copy-pasted from
```

Comment:
Actually, this is very stupid. We need a reliable way to ask for specific symbols, and we need an automatic update script for this derivation: e.g. one that objdumps the current libstdcxx, sorts symbols by versions, and takes the newest stuff.

On that note, I should rework `tests.loadOld`. The fact that it works right now is sheer accident